app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/PlanWizardInstancePropertiesStepReducer.js

```diff
@@ -23,6 +42,52 @@ export default (state = initialState, action) => {
         .set('errorTenantsWithAttributes', action.payload)
         .set('isFetchingTenantsWithAttributes', false)
         .set('isRejectedTenantsWithAttributes', true);
+    case `${QUERY_V2V_OSP_BEST_FIT_FLAVOR}_PENDING`:
+      return state.set('isFetchingBestFitFlavor', true).set('isRejectedBestFitFlavor', false);
+    case `${QUERY_V2V_OSP_BEST_FIT_FLAVOR}_FULFILLED`:
+      return state
+        .set('bestFitFlavors', action.payload.data.results)
+        .set('isFetchingBestFitFlavor', false)
+        .set('isRejectedBestFitFlavor', false)
+        .set('errorBestFitFlavor', null);
+    case `${QUERY_V2V_OSP_BEST_FIT_FLAVOR}_REJECTED`:
+      return state
+        .set('errorBestFitFlavor', action.payload)
+        .set('isFetchingBestFitFlavor', false)
+        .set('isRejectedBestFitFlavor', true);
+    case SET_V2V_BEST_FIT_FLAVORS_AND_DEFAULT_SECURITY_GROUPS: {
+      const vmBestFitFlavors = action.payload;
+      const instancePropertiesRowsUpdatedWithBestFlavor = [];
+      vmBestFitFlavors.forEach(vmFlavor => {
+        const existingInstancePropertiesRow = state.instancePropertiesRows.find(row => row.id === vmFlavor.vm_id);
+        const tenant = state.tenantsWithAttributes.find(
+          tenantsWithAttribute => tenantsWithAttribute.id === vmFlavor.tenant_id
+        );
+        const tenantFlavors = tenant.flavors;
+        const tenantSecurityGroups = tenant.security_groups;
+        const bestFitFlavorName = tenantFlavors.find(flavor => flavor.id === vmFlavor.flavor_id).name;
+        const defaultSecurityGroupName = 'default';
+        const defaultSecurityGroupId = tenantSecurityGroups.find(
+          securityGroup => securityGroup.name === defaultSecurityGroupName
+        ).id;
```

Comment:
We should guard against a `TypeError` here.  `[].find()` returns `undefined` if there is no matching array element